# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/shadow/String.h
# Commit: ae6126131b6d
# Full Hash: ae6126131b6dfa18981f25c6986200c14e63e5f2
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2024-04-11 04:26:26
# Regressor Bug: 1881995
# File Overlap Count: 1
# Description:
#   Bug 1881995 - Implement ForwardedAtoms and create them during atomization r=iain,sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202690
# ==============================================================================

diff -r 4e5967329fdd -r ae6126131b6d js/public/shadow/String.h
--- a/js/public/shadow/String.h	Wed Apr 10 15:47:15 2024 +0000
+++ b/js/public/shadow/String.h	Wed Apr 10 16:32:23 2024 +0000
@@ -34,9 +34,9 @@
   static constexpr uint32_t ATOM_BIT = js::Bit(3);
   static constexpr uint32_t LINEAR_BIT = js::Bit(4);
   static constexpr uint32_t INLINE_CHARS_BIT = js::Bit(6);
-  static constexpr uint32_t LATIN1_CHARS_BIT = js::Bit(9);
+  static constexpr uint32_t LATIN1_CHARS_BIT = js::Bit(10);
   static constexpr uint32_t EXTERNAL_FLAGS = LINEAR_BIT | js::Bit(8);
-  static constexpr uint32_t TYPE_FLAGS_MASK = js::BitMask(9) - js::BitMask(3);
+  static constexpr uint32_t TYPE_FLAGS_MASK = js::BitMask(10) - js::BitMask(3);
   static constexpr uint32_t PERMANENT_ATOM_MASK = ATOM_BIT | js::Bit(8);
 
   uintptr_t flags_;